.github/dependabot.yml

```diff
@@ -0,0 +1,42 @@
+updates:
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+      interval: 'monthly'
+    open-pull-requests-limit: 10
+    commit-message:
+      prefix: 'chore'
+    groups:
+      # Any updates not caught by the group config will get individual PRs
+      gha-low-risk:
+        update-types:
+          - 'minor'
+          - 'patch'
+
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'monthly'
+    open-pull-requests-limit: 10
+    commit-message:
+      prefix: 'chore'
+    ignore:
+      # Prevent updates to ESM-only versions
+      - dependency-name: 'chalk'
+        versions: ['>=5.0.0']
+      - dependency-name: 'esbuild'
+        versions: ['>=0.11.0']
+      - dependency-name: 'execa'
+        versions: ['>=6.0.0']
+      - dependency-name: 'globby'
+        versions: ['>=12.0.0']
+      - dependency-name: 'html-entities'
+        versions: ['>=2.0.0']
+      - dependency-name: 'markdown-table'
```

Comment:
Can we just rip this out instead of pinning it? the one table it's being used to generate is trivial (all values are known statically to not contain special characters), and any changes it generates will be part of a PR and reviewed anyway. The whole dependency could be replaced by something like:

```js
      # Before:
      const attributesTableMarkdown = mdTable([
        headings.attributesMdTableHeader,
        ...attributesTable
      ]);

      # After:
      const formatMarkdownTableRow = (columnValues) => `| ${columnValues.join(' | ')} |`;
      const attributesTableWithHeader = [
        headings.attributesMdTableHeader,
        ['---', '---'],
        ...attributesTable
      ];
      const attributesTableMarkdown = attributesTableWithHeader.map(formatMarkdownTableRow).join('\n');
```